.github/workflows/shared-build-deploy-container.yml

```diff
@@ -128,9 +131,25 @@ jobs:
             fi
           fi
 
+  build-test-cover-solution:
```

Comment:
I don't quite agree of having this job here, since we are already running this in our pull requests. So IMO it is just taking more time to deploy with something we already did. 🤔 

What are your thoughts about this?